astropy/table/table.py

```diff
@@ -630,7 +628,7 @@ def show_in_browser(self,
             the table in memory).  A negative value of ``max_lines`` indicates
             no row limit
         jsviewer : bool
-            If `True`, prepends some javascript headers so that the table is
+            If `True`, pretends some javascript headers so that the table is
```

Comment:
Here, ``prepends`` was correct (opposite of appends - see http://en.wiktionary.org/wiki/prepend)